### ISSUE TITLE
View Result message does not display even when value is overflowing in the cell

### ISSUE THREAD

**Issue by SharathAlur** [cerner/terra-clinical#761]
# Bug Report

## Description

View Result message does not display even when value is overflowing in the cell

## Steps to Reproduce

1. Add a large value, like 23898989899992.5
2. View result message does not display on cell, instead the value is shown with elipsis.
3.
4.

## Additional Context / Screenshots

<img width="1763" alt="Screenshot 2021-06-17 at 10 01 38 PM" src="https://user-images.githubusercontent.com/22268936/122742520-a71b0480-d2a3-11eb-963b-46bc056a7916.png">

## Expected Behavior

Cell should display view result message

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

**Comment by neilpfeiffer** [comment]
@SharathAlur - Please verify your implementation and mock data contains the correct object API for indicating a numeric result value, therefore will apply the "View Results" display when large value exceeds the display space in the flowsheet-cell component.

It should be  `isNumeric: true,` and not `type: 'NUMERIC',`

Current Numeric Overflow example:
https://github.com/cerner/terra-clinical/blob/1b889e41cbeba71743bd19210e94b0c1a70744f1/packages/terra-clinical-result/src/terra-dev-site/doc/example/flowsheet-result-cell/FlowsheetResultCellNumericOverflow.jsx#L10-L20



----
\
\
\
For reference, pre-v.1.0 beta releases of the component originally contained `type: 'NUMERIC',` but was updated from a type enum to a simple bool:
https://github.com/cerner/terra-clinical/pull/574/commits/eb45fd842e6ee34fa355096be0ecf44e54483fd5#diff-1b09cfb2c9ed65173a6fe6991f54aae56766fdc74ac6718bc84198fef29ae79cL55-R55
```diff
- 55     type: 'NUMERIC',
+ 55     isTypeNumeric: true,
```

It was then updated again for final before the v1.0 release with the `isNumeric` key:
https://github.com/cerner/terra-clinical/commit/51ee5012627534065b337cb2891d543ecb7671b1#diff-1b09cfb2c9ed65173a6fe6991f54aae56766fdc74ac6718bc84198fef29ae79cL79-R79
```diff
- 55     isTypeNumeric: true,
+ 55     isNumeric: true,
```

**Comment by jmsv6d** [comment]
@SharathAlur see comment on #762

**Comment by SharathAlur** [comment]
Thanks, This is resolved after setting isNumeric to true.

**Comment by SharathAlur** [comment]
This issue can be closed

**Comment by ryanthemanuel** [comment]
Closing per the comments above.
